server/internal/infrastructure/mongo/migration/240515221450_copy_scene_id.go

```diff
@@ -0,0 +1,98 @@
+package migration
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/reearth/reearth/server/internal/infrastructure/mongo/mongodoc"
+	"github.com/reearth/reearth/server/pkg/id"
+	"github.com/reearth/reearthx/log"
+	"github.com/reearth/reearthx/mongox"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type seed struct {
+	projectId id.ProjectID
+	sceneId   id.SceneID
+}
+
+func CopySceneId(ctx context.Context, c DBClient) error {
+	col := c.WithCollection("scene")
+
+	var seeds []seed
+
+	col.Find(ctx, bson.M{}, &mongox.BatchConsumer{
```

Comment:
Instead of just logging errors, consider returning them from the CopySceneId function. This will allow the caller to handle errors appropriately.

Reply:
fixed in [634a084](https://github.com/reearth/reearth/pull/981/commits/634a084feb25aa4580d18bcf3c8fe0a28ff771c0)